hack/unit-tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+source ./hack/go-get-tool.sh
+
+REPO_ROOT=$(realpath "$(dirname "${BASH_SOURCE[0]}")/..")
+
+ENVTEST_VERSION=v0.7.0
+ENVTEST_ASSETS_DIR=/tmp/testbin
+ENVTEST_SETUP_SCRIPT=https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/${ENVTEST_VERSION}/hack/setup-envtest.sh
+
+function setupEnvtest() {
+  echo "Envtest version: ${ENVTEST_VERSION}."
+  mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh ${ENVTEST_SETUP_SCRIPT}
+	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh
+	fetch_envtest_tools ${ENVTEST_ASSETS_DIR}
+	setup_envtest_env ${ENVTEST_ASSETS_DIR}
```

Comment:
Normally you use 2 spaces for indentation, but here we have tabs. could you fix this for consistency?

Reply:
made it consistent with other sh files there. Didn't noticed that it uses 4 spaces.